content/300-guides/050-database/400-prototyping-schema-db-push.mdx

```diff
@@ -152,71 +152,71 @@ The following scenario demonstrates how to use `db push` to prototype a change t
 
 1. Check out the latest Prisma schema and migration history:
 
-    ```prisma
-    generator client {
-        provider = "prisma-client-js"
-    }
-
-    datasource db {
-        provider = "postgresql"
-        url      = env("DATABASE_URL")
-    }
-
-    model User {
-        id       Int      @id @default(autoincrement())
-        name     String
-        jobTitle String
-        posts    Post[]
-        profile  Profile?
-    }
-
-    model Profile {
-        id       Int    @id @default(autoincrement())
-        biograpy String // Intentional typo!
-        userId   Int
-        user     User   @relation(fields: userId, references: id)
-    }
-
-    model Post {
-        id         Int        @id @default(autoincrement())
-        title      String
-        published  Boolean    @default(true)
-        content    String     @db.VarChar(500)
-        authorId   Int
-        author     User       @relation(fields: [authorId], references: [id])
-        categories Category[]
-    }
-
-    model Category {
-        id    Int    @id @default(autoincrement())
-        name  String @db.VarChar(50)
-        posts Post[]
-        @@unique([name])
-    }
-    ```
+   ```prisma
+   generator client {
+       provider = "prisma-client-js"
+   }
+
+   datasource db {
+       provider = "postgresql"
+       url      = env("DATABASE_URL")
+   }
+
+   model User {
+       id       Int      @id @default(autoincrement())
+       name     String
+       jobTitle String
+       posts    Post[]
+       profile  Profile?
+   }
+
+   model Profile {
+       id       Int    @id @default(autoincrement())
+       biograpy String // Intentional typo!
+       userId   Int
+       user     User   @relation(fields: userId, references: id)
+   }
+
+   model Post {
+       id         Int        @id @default(autoincrement())
+       title      String
+       published  Boolean    @default(true)
+       content    String     @db.VarChar(500)
+       authorId   Int
+       author     User       @relation(fields: [authorId], references: [id])
+       categories Category[]
+   }
+
+   model Category {
+       id    Int    @id @default(autoincrement())
+       name  String @db.VarChar(50)
+       posts Post[]
+       @@unique([name])
+   }
+   ```
```

Comment:
Sure the indentation of the code block still works in the rendered view, that it is _inside_ the list vs on the same level?

Reply:
Man, these prettier formatting things are super annoying! On the plus side, the indentation doesnt affect its rendered output